workers/unity/Packages/com.improbable.gdk.querybasedinteresthelper/InterestQuery.cs

```diff
@@ -0,0 +1,83 @@
+using System.Collections.Generic;
+
+namespace Improbable.Gdk.QueryBasedInterest
+{
+    /// <summary>
+    ///     Utility class to help construct ComponentInterest.Query objects.
+    /// </summary>
+    public class InterestQuery
```

Comment:
You need to hide the default constructor on this as I can currently do `new InterestQuery()`